comms.js

```diff
@@ -37,8 +37,8 @@ Comms.prototype.write = function(msg) {
 	}
 };
 
-Comms.prototype.listen = function(port) {
-	var self = this, server = this.server = jot.createServer(port);
+Comms.prototype.listen = function(port, opts) {
+	var self = this, server = this.server = jot.createServer(opts || {}, port);
```

Comment:
NM, just read the [net docs](https://nodejs.org/api/net.html#net_net_createserver_options_connectionlistener)